packages/musecore.rb

```diff
@@ -0,0 +1,38 @@
+require 'buildsystems/cmake'
+
+class Musecore < CMake
+  description 'MuseScore is an open source and free music notation software.'
+  homepage 'https://musescore.org'
+  version '4.3.2'
+  license 'GPL-3'
+  compatibility 'x86_64 aarch64 armv7l'
+  source_url 'https://github.com/musescore/MuseScore.git'
+  git_hashtag "v#{version}"
+  binary_compression 'tar.zst'
+
+  binary_sha256({
+    aarch64: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+     armv7l: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+     x86_64: 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
```

Comment:
Missing binaries?

Reply:
As I said in the PR description-- these need to be tested with gui.

Reply:
You can still provide the binaries for testing.

Reply:
The tests for the package fail, so the build doesn't compete and I can't upload binaries.

Reply:
Well, then more work needs to be put in so the tests don't fail.

Reply:
The tests fail because there is no gui, hence why I am asking for someone with gui to build and test.